i18n/en/docusaurus-plugin-content-docs/current/about/understanding/naming.md

```diff
@@ -4,59 +4,60 @@ sidebar_position: 4
 
 # Naming
 
-## Problem
+Different developers have different experiences and contexts, which can lead to misunderstandings on the team when the same entities [are called differently][disc-src]. For example:
 
-Due to the fact that each developer has his own experience and development context - [we are used to calling the same entities differently][disc-src], which can lead to misunderstandings within the team.
+- Components for display can be called "ui", "components", "ui-kit", "views", ...
+- The code that is reused throughout the application can be called "core", "shared", "app", ...
+- Business logic code can be called "store", "model", "state", ...
 
-- *Components to be displayed - `ui` / `components` / `ui-kit` / `views` / ...*
-- *Code reusable in all parts of the application - the `core`/ `shared` / `app` / ...*
-- *Code business logic `store` / `model` / `state` / ...*
+## Naming in Feature-Sliced Design {#naming-in-fsd}
 
-## Naming in FSD
+The methodology uses specific terms such as:
 
-The methodology uses such terms as
+- [Layers][refs-layers] — "app", "process", "page", "feature", "entity", "shared"
+- [Slices][refs-slices]
```

Comment:
issue: I don't think this item belongs here. The original intention was to highlight that there were some "reserved" keywords among layers and segments, the slices don't have them.

suggestion: 
```markdown
- "app", "process", "page", "feature", "entity", "shared" as layer names
- "ui', "model", "lib", "api", "config" as segment names
```

Reply:
Yes, that looks good